### DISCUSSION TITLE
Information process

### DISCUSSION THREAD

**Discussion by CristianSVV** [community/community#38999] [GitHub Education]
I am requesting GitHub for students, does it take a long time to process the information? Because I've been going for almost an hour and it still hasn't passed this part.
![Capture](https://user-images.githubusercontent.com/118090174/201528166-00c9d9de-8143-4044-a768-fcf5e1b2bea0.jpg)

**Comment by samus-aran** [comment]
Hi @CristianSVV 👋 

Sorry that it has taken a long time to process this page. How did you get on? Was it able to be submitted?
